securedrop/bin/dev-shell

```diff
@@ -61,7 +61,15 @@ function docker_run() {
 
     # If this is a CI run, pass CodeCov settings into the container.
     if [ -n "${CIRCLE_BRANCH:-}" ] ; then
-        ci_env=$(bash <(curl -s https://codecov.io/env))
+        tmpdir=$(mktemp -d -t codecov-XXXX)
+        curl -s https://codecov.io/bash > "$tmpdir"/codecov; # env isn't in SHA256SUM yet!!
+        curl -s https://codecov.io/env > "$tmpdir"/env;
+        VERSION=$(curl --silent "https://api.github.com/repos/codecov/codecov-bash/releases/latest" | grep '"tag_name":' |sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
Suggestion: `curl -s https://api.github.com/repos/codecov/codecov-bash/releases/latest | jq -r .tag_name` for posterity's sake.